index.js

```diff
@@ -1,7 +1,7 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import dotenv from 'dotenv';
-import routes from './routes/index';
+// import routes from './routes/index';
```

Comment:
I'm not sure it's the best idea to have commented code in a PR. In any case, is this unnecessary? Won't it still come up when we write routes? If it will then it would be best to leave it as is till we use it.